winterfell/webapp/src/templates/claimSelectForms.pug

```diff
@@ -9,15 +9,15 @@
         uib-progressbar.progressbar(value="numRequiredCompleted", max="numRequiredForms", type="{{numRequiredCompleted == numRequiredForms ? 'success' : 'info'}}")
           span {{numRequiredCompleted}}/{{numRequiredForms}}
 
-      div.form-wrapper(ng-repeat="(formId, form) in forms")
+      div.form-wrapper(ng-repeat="(formId, form) in allForms", ng-init="myForm = myForms[formId]")
         div.form-content
-          div.state-wrapper(ng-class="{'required': formTemplates[form.key].required, 'complete': form.answered == form.answerable}")
+          div.state-wrapper(ng-class="{'required': form.vfi.required, 'complete': myForm && myForm.answered > 0 && myForm.answered == myForm.answerable}")
             div.state-icon
           div.form-title-wrapper
-            h5.title(ng-class="{'required': formTemplates[form.key].required}") {{form.id}}
-            h5.warning.required(ng-if="form.state == formTemplates[form.key].required") &nbsp;(required)
-            h5.subtitle {{formTemplates[form.key].unofficialTitle}}
-          div.form-summary {{formTemplates[form.key].unofficialDescription}}
+            h5.title(ng-class="{'required': form.vfi.required}") {{formId}}
+            h5.warning.required(ng-if="form.vfi.required && !(myForm && myForm.answered > 0 && myForm.answered == myForm.answerable)") &nbsp;(required)
+            h5.subtitle {{form.vfi.title}}
+          div.form-summary {{form.vfi.summary}}
         div.form-buttons
           button.edit-btn(ui-sref="root.form({claimId: claimId, formId: formId})")
```

Comment:
does this link still work? Shouldn't formId now be the name of the form and not the mongo identifier?